policyengine_us/variables/gov/states/mi/tax/income/deductions/standard/tier_two/mi_standard_deduction_tier_two_increase_eligible.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class mi_standard_deduction_tier_two_increase_eligible(Variable):
```

Comment:
in general `eligible` by itself should be a bool. do the same for the parallel vars
```suggestion
class mi_standard_deduction_tier_two_increase_eligible_count(Variable):
```